tlcc_enclave/Makefile

```diff
@@ -6,12 +6,16 @@ TOP = ..
 include $(TOP)/build.mk
 
 PROTO_BUILD_DIR := enclave/protos
+FABRIC_PROTOS := protos/fabric
 BUILD_DIR := _build
 
 .PHONY: protos
 protos: $(PROTO_BUILD_DIR)
 
-$(PROTO_BUILD_DIR):
+$(FABRIC_PROTOS):
+	git clone https://github.com/hyperledger/fabric-protos.git $@
```

Comment:
I hesitantly suggest fabric-protos as a submodule, but if this works, I'm fine with it.

Reply:
we use modules now :D